Dockerfile

```diff
@@ -5,14 +5,12 @@ ENV NODE_ENV production
 EXPOSE 5000
 
 RUN apt-get update \
-  && apt-get -y install cron supervisor nano --no-install-recommends \
+  && apt-get -y install supervisor nano --no-install-recommends \

```

Comment:
@mramato I assume this is all that needs to be removed from the Dockerfile here if we're removing `cron`?

Reply:
Also do you remember if there was a reason we didn't go with a node-based approach and opted from cron in the first place?

Reply:
I don't know why this was the original choice since that was before my time, but the reason I stuck with cron and tried to fix it was I thought it would be quicker than replacing it.

Reply:
Sorry, that was a question for @mramato :laughing:

Reply:
Cron was always a bad choice supervisor isn't really needed either.  I was just a crappy mentor to the intern we had working on it and didn't have time to find him a better solution.

Using Node directly is technically a bad choice too, but will work since we only ever run a single concierge server.  If we ever had multiple concierges, we would need to do something different (though I haven't really looked at this PR in general).

Reply:
Right, but an init process isn't needed for Docker containers.  It's the system running the container that will worry about restarting it if there's a problem or anything like this.

Not a change that needs to be made now, but the only reason the supervisor was there was because of cron.